src/olympia/amo/tests/__init__.py

```diff
@@ -895,11 +895,6 @@ def setUpClass(cls):
         # right before each test.
         stop_es_mocks()
         cls.es = amo_search.get_es(timeout=settings.ES_TIMEOUT)
-        cls._SEARCH_ANALYZER_MAP = amo.SEARCH_ANALYZER_MAP
-        amo.SEARCH_ANALYZER_MAP = {
```

Comment:
This has been here forever and was actively harmful to relevancy score testing in other languages. In fact, besides the new tests added around `en-CA` and `en-GB`, this is the only reason why I've had to update the expected scores in `test_search_ranking` : otherwise, my changes wouldn't affect them at all (which makes sense because none of the ranking tests used languages that had the same analyzer)